src/System.Net.Requests/tests/HttpWebRequestHeaderTest.cs

```diff
@@ -25,12 +29,12 @@ public void Ctor_VerifyHttpRequestDefaults(Uri remoteServer)
             Assert.False(request.HaveResponse);
             Assert.NotNull(HttpWebRequest.DefaultCachePolicy);
             Assert.Null(request.CookieContainer);
-            Assert.Equal(0,HttpWebRequest.DefaultMaximumErrorResponseLength);
-            Assert.Equal(65536,HttpWebRequest.DefaultMaximumResponseHeadersLength);
+            Assert.Equal(0, HttpWebRequest.DefaultMaximumErrorResponseLength); // NetFX behavior difference (64 on NetFX).
+            Assert.Equal(65536, HttpWebRequest.DefaultMaximumResponseHeadersLength); // NetFX behavior difference (64 on NetFX).
             Assert.Null(request.CookieContainer);
             Assert.True(request.AllowWriteStreamBuffering);
             Assert.NotNull(request.ClientCertificates);
-            Assert.Throws<NotImplementedException>(() => request.ConnectionGroupName);
+            Assert.Throws<NotImplementedException>(() => request.ConnectionGroupName); //NetFX behavior difference.
```

Comment:
Should these behavioral differences be separated out into a separate test attributed with SkipOnTarget for NetFramework?